CHANGELOG.md

```diff
@@ -1 +1,30 @@
+# 0.6.0
+
+## Prelude
+
+Added new features and commands, such as:
+
+- 'subgit delete'
+- 'subgit import'
+- 'subgit reset'
+- 'subgit clean'
+
+## General fixes
+
+- Added error handling for 'subgit pull'.
+	- If subgit config file has a repo with `branch: `, the command exits
+
+- Implemented multiprocessing for 'subgit fetch' commnand. [#41][https://github.com/dynamist/subgit/pull/41]
```

Comment:
You have this PR41 in both this general and new features. Each PR should only be in one block